File: Java_Programming/Practice/dynamic_memory_allocation.java
import java.util.Scanner;
public class DynamicMemory
{
  public static void main(String[]args) 
   {
     Scanner sc=new Scanner(System.in);
     System.out.println("Enter number of elements :");
     int n =sc.nextInt();
     int[] arr=new int[n];
     System.out.println("Enter" + n +"elements:");
     for(int i=0;i<n;i++)
     {
       arr[i]=sc.nextInt();
     }
     System.out.println("Elements are:");
     for(int i=0;i<n;i++)
     {
       System.out.println(arr[i]+" ");
     }
     sc.close();
   }
}

     

File: C++_Programming/Practice/dynamic_memory_allocation.cpp
#include<iostream>
using namespace std;
int main()
{
   int n;
   cout<<"Enter number of elements :";
   cin>>n;
  
  int *arr=new int[n];
  cout<<"Enter "<<n<<"elements:\n"; 
  for(int i=0;i<n;i++)
  {
     cin>>arr[i];
  }
  cout<<"Elements are :\n";
  for(int i=0;i<n;i++)
  {
    cout<<arr[i]<<"";
  }
  delete[]arr;
  return 0;
}